soccer/src/referee/InternalReferee.hpp

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include <rclcpp/rclcpp.hpp>
+#include <rj_msgs/srv/quick_commands.hpp>
+#include <rj_msgs/srv/quick_restart.hpp>
+
+#include "RefereeBase.hpp"
+#include "config_client/config_client.h"
+
+namespace referee {
+
+class InternalReferee : public RefereeBase {
```

Comment:
Wait what's internal referee?

Reply:
Described in PR body. Handles quick commands (AKA start/stop/halt from inside soccer)

Reply:
A docstring in code would be nice. Nobody is gonna look at the PR body to tell what a class does.